notebooks/114-simplified-mode/114-simplified-mode.ipynb

```diff
@@ -0,0 +1,179 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "POT Simplified Mode tutorial"
```

Comment:
I propose adding some short intro about the Simplified mode which works with CV only and doesn't guarantee an acceptable accuracy drop.